lib/web_ui/lib/src/engine/canvaskit/canvaskit_api.dart

```diff
@@ -2167,6 +2167,16 @@ extension SkLineMetricsExtension on SkLineMetrics {
   external double get lineNumber;
 }
 
+@JS()
+@anonymous
+@staticInterop
+class SkRectWithDirection {}
+
+extension SkRectWithDirectionExtension on SkRectWithDirection {
+  external Float32List rect;
```

Comment:
Does the meaning of elements change based on the value of `direction`? E.g. is is always LTRB, or can it also be RTLB? Might be worth leaving a comment explaining it.

Reply:
Rectangles are always LTRB